lib/web_ui/dev/chrome.lock

```diff
@@ -1 +1,4 @@
-695653
```

Comment:
Now that this is a proper yaml file, should it be `chrome_lock.yaml` or `chrome_version.yaml`? To me a lock file is normally something that you "touch" in the file system to prevent race conditions and stuff like that.

Reply:
👍